README.md

```diff
@@ -57,9 +57,10 @@ Returns a new focus trap on `element`.
 - **onActivate** {function}: A function that will be called when the focus trap activates.
 - **onDeactivate** {function}: A function that will be called when the focus trap deactivates,
 - **initialFocus** {element|string}: By default, when a focus trap is activated the first element in the focus trap's tab order will receive focus. With this option you can specify a different element to receive that initial focus. Can be a DOM node or a selector string (which will be passed to `document.querySelector()` to find the DOM node).
+- **fallbackFocus** {element|string}:
```

Comment:
"Left description in README empty so that you can propose what to write there."

I think it will be better if you will write description as you like.
You can write it here and i will amend it to the pull.